docs/release_notes.adoc

```diff
@@ -57,12 +60,25 @@ spockPull:1910[]
 will now require that the spec is annotated with <<parallel_execution.adoc#isolated-execution, @Isolated>> or `@ResourceLock(org.spockframework.runtime.model.parallel.Resources.META_CLASS_REGISTRY)`. See <<interaction_based_testing.adoc#global-mocks-parallel-execution, Global mocks and parallel execution>> spockPull:1848[]
 * `@TempDir` `spock.tempDir.keep` has been replaced by `spock.tempdir.cleanup`. See <<extensions.adoc#temp-dir-cleanup,TempDir Cleanup>> spockPull:1525[]
 
+////
+
+== 2.4-M7 (tbd)
+
+=== Highlights
+
+* Spock supports Groovy 5.0 spockIssue:2196[]
+* Spock is also tested to run correctly on Java 25 LTS spockPull:2212[]
+
 === Misc
 
+* Fix handling of condition method calls spockPull:2162[]
 * Fix vararg handling in SpyStatic spockIssue:2161[]
 * Fix incompatibility with JUnit 6 in OSGi environment spockIssue:2231[]
 * OSGi: Pin the `Require-Capability:osgi.ee=JavaSE` to Java `8` spockPull:2233[]
 * Fix: Prevent NPE in SpecRunHistory.sortFeatures when duration is missing spockIssue:2234[]
+* Move spec internals methods to be static method calls spockPull:2209[]
+
+Thanks to all the contributors to this release: Andreas Turban, Björn Kautler, Christoph Loy, Leonard Brünings, Thanos Tsiamis
```

Comment:
Judging by the other version sections, Leonard is usually not listing himself as contributor 🤷‍♂️